ulib/FStar.ReflexiveTransitiveClosure.fsti

```diff
@@ -0,0 +1,16 @@
+module FStar.ReflexiveTransitiveClosure
+
+open FStar.Preorder
+
+val closure (#a:Type0) (r:relation a) : preorder a
+
+val closure_step: #a:Type0 -> r:relation a -> x:a -> y:a
+  -> Lemma (requires r x y) (ensures closure r x y)
+
+val closure_inversion: #a:Type0 -> r:relation a -> x:a -> y:a
+  -> Lemma (requires closure r x y)
+          (ensures  x == y \/ r x y \/ (exists z. closure r x z /\ closure r z y))
```

Comment:
Could another, single-step form of the inversion lemma also be useful? 
i.e., the last disjunct is `exists z. r x z /\ closure r z y`

Reply:
I simplified it to `(ensures  x == y \/ (exists z. r x z /\ closure r z y))`